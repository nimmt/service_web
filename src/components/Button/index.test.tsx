import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import { SubmitButton } from './index';

it('renders correctly', () => {
  const tree = ReactTestRenderer
    .create(<SubmitButton value="å€¤" />)
    .toJSON()

  expect(tree).toMatchSnapshot()
});
