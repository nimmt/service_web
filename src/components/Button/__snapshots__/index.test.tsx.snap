// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<input
  type="submit"
  value="å€¤"
/>
`;
